ci/create_initial_account.py

```diff
@@ -21,7 +21,7 @@ async def insert(tx):
             ('creating', username, email, 1, 0),
         )
 
-    return await insert(db)
+    return await insert()
```

Comment:
You might need to add this pylint message (used in batch/front_end.py) `# pylint: disable=no-value-for-parameter`

Reply:
Looks like files not in `ci/ci` don't get checked by pylint. I can do a separate pr to address all the linting issues?

Reply:
Yes, let's get this in. That's definitely a bigger project.